code/modules/background/citizenship/ipc.dm

```diff
@@ -65,5 +65,6 @@
 	allowing its populace to work, study, and travel abroad in the Coalition and beyond, primarily in the All-Xanu Republic, utilizing Xanan documents."
 
 	job_species_blacklist = list(
-		"Consular Officer" = ALL_SPECIES
+		"Consular Officer" = ALL_SPECIES,
+		"Diplomatic Aide" = ALL_SPECIES
```

Comment:
```suggestion
		"Diplomatic Aide" = ALL_SPECIES,
```

https://github.com/Aurorastation/Aurora.3/blob/master/.github/guides/STYLE.md#multi-lining

And it should be good to go